### ISSUE TITLE
Averaging Method (difference in numbers of Schools, private vs public) 

### ISSUE THREAD

**Issue by robilizando** [UBC-MDS/DSCI_522_Group304#25]
Anny raised the following question when working on EDA file to avoid double counting the schools:

" if we simply average all the averages, we’re not considering the *number of students* each average applies to when we’re comparing groups (for example, we’re comparing mean scores for all independent schools vs public schools, but what if there are 1000x more public school students than independent school students? Each school is weighted evenly, when maybe they shouldn’t be)."

What should we do?

**Comment by robilizando** [comment]
Don't we have the numbers of students that took the test per school?. I think we do.. If so we could do the weighted avg that you mentioned to get a better estimate, and score comparison.

**Comment by annychih** [comment]
I spoke with Firas about this and it seems like this may simply be a limitation of our dataset. We can't duplicate the scores so we have a dataset with a row per student because that would assume that all students of a subgroup in a specific school all have the same FSA score. We also can't assume a normal distribution of scores and assign scores randomly from that distribution because we don't know what the variance would be etc. 

So, in short: we're basically ignoring the number_writers column and we'll have to note this limitation of our data somewhere in the report.
